ember/app/templates/not-found.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="not-found" data-test-not-found>
+  <h1>404</h1>
+  <p>The site you requested does not exist</p>
+</div>
```

Comment:
This will result in the exact same styling as before but without custom CSS:

```suggestion
<div class="uk-text-center" data-test-not-found>
  <h1 class="uk-heading-2xlarge">404</h1>
  <p>The site you requested does not exist</p>
</div>
```

Reply:
done here https://github.com/adfinis/Outdated/pull/72/commits/97a99b16d5b65943b98f21f70f841b1096ff6202